new/LIT6956MalkeaMadAlam.xml

```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6956MalkeaMadAlam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Malkǝʾa Madḫane ʿAlam - ʾƎsaggǝd</title>
```

Comment:
Please shape the title according to other malke' records, cp. LIT1867Malkea or LIT6158MalkeaGalawdewos (the second one seems to be a more recent muster). Somewhere (in abstracts or `<title xml:lang="gez" xml:id="t2">`) the date 27 Maggabit can be mentioned, and encoded in the text below  as `<date/>`.

Reply:
Should this date be specified or is it enough to encode `<date>መጋቢት፡ ፳፯፡</date>`?

Reply:
It can be specified in abstract, or you can make t2 "Image to the Saviour of the World for 27 Maggabit"